src/design-tokens/_easing.yml

```diff
@@ -25,11 +25,11 @@ aliases:
   ease_in_out_back: cubic-bezier(0.68, -0.55, 0.265, 1.55)
 props:
   ease_in_out:
-    value: "{!ease_in_out_quad}"
+    value: '{!ease_in_out_quad}'
```

Comment:
These changes in .yml files are because now prettier is being run on all files it supports